cli/configuration.schema.js

```diff
@@ -0,0 +1,59 @@
+export default {
+  $schema: 'http://json-schema.org/schema#',
+  title: 'Configuration',
```

Comment:
Nit-picky: how about we rename this to `Yeep configuration` for better AJV info/error messages.

Reply:
done